### PR TITLE
Update notification pattern for vanilla 3.0 in k8s docs

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

- The sequel to: https://github.com/canonical-web-and-design/ubuntu.com/pull/11310
- The notification pattern was updated, and support for the old version of the pattern is removed in Vanilla 3.x, this PR updates many instances across the project to the new structure, mostly within the k8s docs - see the upgrade guide: https://vanillaframework.io/docs/migration-guide-to-v3#notifications
- Note: running a find and replace on a couple of the classnames automatically linted some of these files, adding extra noise to the review...

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-11314.demos.haus/
- Check the files that were changed, and find the pages on the demo, see that the notifications on them are not broken

## Issue / Card

(partially) fixes https://github.com/canonical-web-and-design/web-squad/issues/4854

### CODE REVIEW

File: templates/kubernetes/docs/1.16/upgrading.md [hunk 1]
```diff
@@ -217,13 +214,13 @@ juju config kubernetes-master channel=1.10/stable
...
-  <p markdown="1" class="p-notification__response">
-    <span class="p-notification__status">Note:</span>
-Once the configuration has been changed, the charms will be put into a `blocked` state.
-You must continue the upgrade process, even if you revert the configuration to the
-currently active version of Kubernetes.
-  </p>
+<div class="p-notification--caution is-inline">
+  <div markdown="1" class="p-notification__content">
+    <span class="p-notification__title">Note:</span>
+    <div class="p-notification__message">Once the configuration has been changed, the charms will be put into a `blocked` state.
```
Comment (petesfrench) [inline templates/kubernetes/docs/1.16/upgrading.md:220]:
```suggestion
    <div class="p-notification__message">Once the configuration has been changed, the charms will be put into a `blocked` state.</div>
```
The rest of the page was being wrapped in this notification

Comment (sowasred2012) [reply templates/kubernetes/docs/1.16/upgrading.md:220]: Oops!

File: templates/kubernetes/docs/1.17/upgrading.md [hunk 2]
```diff
@@ -183,15 +180,15 @@ example, if you are using the flannel CNI:
...
-Some services may be briefly interrupted during the upgrade process. Upgrading
-your CNI (e.g. flannel) will cause a small amount of network downtime. Upgrading
-<strong>easyrsa</strong> will not cause any downtime. The behaviour of other
-components you have added to your cluster may vary - check individual documentation
-for these charms for more information on upgrades.
-  </p>
+<div class="p-notification--caution is-inline">
+  <div markdown="1" class="p-notification__content">
+    <span class="p-notification__title">Note:</span>
+    <p class="p-navigation__message">Some services may be briefly interrupted during the upgrade process. Upgrading
```
Comment (petesfrench) [inline templates/kubernetes/docs/1.17/upgrading.md:186]: Not sure why, but it doesn't appear inline in the demo

File: templates/kubernetes/docs/1.19/upgrading.md [hunk 3]
```diff
@@ -283,12 +280,12 @@ juju config kubernetes-master channel=1.19/stable
...
-Once the configuration has been changed, the charms will be put into a `blocked` state.
-You must continue the upgrade process, even if you revert the configuration to the
-currently active version of Kubernetes.
+<div class="p-notification--caution is-inline">
+  <div markdown="1" class="p-notification__content">
+    <span class="p-notification__title">Note:</span>
+    <p class="p-notification__message">Once the configuration has been changed, the charms will be put into a `blocked` state.
+    You must continue the upgrade process, even if you revert the configuration to the
+    currently active version of Kubernetes.</p>
   </p>
```
Comment (petesfrench) [inline templates/kubernetes/docs/1.19/upgrading.md:289]:
```suggestion
  </div>
```
Big notification box again

File: templates/kubernetes/docs/backups.md [hunk 4]
```diff
@@ -72,10 +75,13 @@ sha256sum etcd-snapshot-2018-09-26-18.04.02.tar.gz
...
-<span class="p-notification__status">Warning:</span>
-Restoring a snapshot should not be performed when there is more than one unit of **etcd** running.
- </p></div>
+<div class="p-notification--caution is-inline">
+  <div markdown="1" class="p-notification__content">
+    <span class="p-notification__title">Warning:</span>
+    <p class="p-notification__message">Restoring a snapshot should not be performed when there is more than one unit of **etcd** running.
+  </p>
+ </div>
+</div>
```
Comment (petesfrench) [inline templates/kubernetes/docs/backups.md:85]: The feedback notification on this page needs updating

File: templates/kubernetes/docs/quickstart.md [hunk 5]
```diff
@@ -33,9 +33,12 @@ Kubernetes cluster running on the cloud of your choice in minutes!
...
-
+<div class="p-notification--positive is-inline">
+  <div markdown="1" class="p-notification__content">
+    <span class="p-notification__title">Note:</span>
+    <p class="p-notification__message">If you don't meet these requirements, there are additional ways of installing <emphasis>Charmed Kubernetes<sup>&reg;</sup></emphasis>, including additional OS support and an entirely local deploy. Please see the more general <a href="/kubernetes/docs/install-manual">Installing Charmed Kubernetes</a> page for details. </p>
+  </div>
+</div>
 
 <section class="p-strip--light is-bordered">
   <div class="u-fixed-width">
```
Comment (petesfrench) [inline templates/kubernetes/docs/quickstart.md:44]: I know this isn't from your PR but this stepped list section looks a bit broken

File: templates/kubernetes/docs/vsphere-integration.md [hunk 6]
```diff
@@ -113,9 +110,10 @@ juju config vsphere-integrator \
 ```
 
 <div class="p-notification--information">
-  <p class="p-notification__response">
-    Note: If any configuration option is set, they must all be set.
-  </p>
+  <div class="p-notification__content">
```
Comment (petesfrench) [inline templates/kubernetes/docs/vsphere-integration.md:113]:
```suggestion
  <div class="p-notification__content is-inline">
```

**Review by petesfrench — CHANGES_REQUESTED** [review]
All good, just a couple things need changing

File: templates/kubernetes/docs/1.17/upgrading.md (hunk 2 above)
Comment (sowasred2012) [reply templates/kubernetes/docs/1.17/upgrading.md:186]: Argh - typo

**Comment by sowasred2012** [comment]
Cheers @petesfrench - all updated

**Review by petesfrench — APPROVED** [review]
LGTM!

**Pull request merged**